### PR TITLE
Created And Updated the readme.md file

### PR DESCRIPTION
**Pull request by keshav2410garg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- The user will be able to perform both basic and scientific calculations using this calculator. For example "+" Addistion "-" Subraction Multiplication "X" aswell as scientific calcution in trignometric operations.</br>
+- The calculator has memory feature. In this feature at a time user will be able to store all the previous calculations and results performend and look into that later.</br>
+
+**HOW IT WORKS:** </br>
+ - The user will enter the input number for which He needs to perform calculation.
+
+**SET UP:**</br>
+1. Fork and clone the repository</br>
+
+     git clone https://github.com/keshav2410garg/scientific-calculator.git
```
Comment (akash2237778) [inline README.md:18]: change URL for upstream i.e. https://github.com/upes-open/scientific-calculator

**Review by akash2237778 — CHANGES_REQUESTED** [review]
Not up to the mark! Please follow README guidlines use [this](https://github.com/upes-open/graph-generator/blob/main/README.md) as a reference. 
Documentation plays a very important role in every project. Specifically for `README.md` it becomes even more important as it's the first impression to the project. I hope you understand! :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/Naereen/StrapDown.js/graphs/commit-activity)
+[![Open Source Love svg2](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
+</br>
+<img src="https://img.shields.io/badge/C%2B%2B-00599C?style=for-the-badge&logo=c%2B%2B&logoColor=white" />
+
+
+
+
+**OVERVIEW:**</br>
+Ever wondered of making you own calculator that will be based your understading of the first language the you learned that is C++ and basic concepts of OOP's programming.
```
Comment (akash2237778) [inline README.md]:
```suggestion
Ever wondered of making your own calculator that will be based on your understanding of the first language you learned that is C++ and basic concepts of OOP's programming.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+So this repository focuses on making a calculator on C++ that will not enhance you C++ skills but also be based on object oriented and how these concepts can be modeled and replicated with
```
Comment (akash2237778) [inline README.md]:
```suggestion
So this repository focuses on making a calculator using C++ that will not only enhance your C++ skills but also be based on object-oriented and how these concepts can be modeled and replicated with 
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+</br>
+<img src="https://img.shields.io/badge/C%2B%2B-00599C?style=for-the-badge&logo=c%2B%2B&logoColor=white" />
+
+
+
+
+**OVERVIEW:**</br>
+Ever wondered of making you own calculator that will be based your understading of the first language the you learned that is C++ and basic concepts of OOP's programming.
+So this repository focuses on making a calculator on C++ that will not enhance you C++ skills but also be based on object oriented and how these concepts can be modeled and replicated with 
+real world. Tis will also make use of basic data stuctures to store the results.
```
Comment (akash2237778) [inline README.md]:
```suggestion
real world. This will also make use of basic data structures to store the results.
```

**Review by akash2237778 — CHANGES_REQUESTED** [review]
@khushboo-jakhmola can you please help us to improve this README? Language is not up to the mark. It doesn't sound technical README.

**Review by khushboo-jakhmola — CHANGES_REQUESTED** [review]
* Please change the case of the title to upper case.

Remove *overview* as a heading and start with the introduction to your project.
"Ever wondered what it takes to build your own calculator? Well, what if we tell you that all you need is a programming language like C++ and some basic knowledge of Object Oriented Programming. Still seems unreal? We'll walk you through it. This repository will not only brush up your knowledge of C++  but also educate you about concepts of OOPs and how they can be modeled and replicated with real world examples."

## FEATURES:
* The user will be able to perform both basic and scientific calculations using this calculator. For example:  "+" Addition, "-" Subtraction, "X" Multiplication,"/" Division and Trignometric operations.
* The calculator comes with an add on memory feature. With the help of this feature user can  store previous calculations and their results which can be referred to later for ease in calculations.
 

*REMOVE* how it works heading and its content -totally unnecessary 

*Few points to keep in mind:*
* Double check your spellings. 
* Highlight wherever its needed with bold and italics.
*  remove the background color as it distracts from the write up. 
* Maintain uniformity in your write up
@keshav2410garg 
@akash2237778

**Comment by akash2237778** [comment]
Looks good!
Thanks @keshav2410garg @khushboo-jakhmola

**Review by khushboo-jakhmola — COMMENTED** [review]
Looks good to me :)

**Pull request merged**